### PR TITLE
CMS 301 changelist

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-301
- Updates Changelist styles

screen shot 2015-08-31 at 15 02 38
screen shot 2015-08-31 at 15 53 11

### CODE REVIEW

File: djangocms_admin_style/sass/partials/_changelist.scss [hunk 1]
```diff
@@ -225,6 +249,21 @@
     }
 }
 #changelist-search {
+    max-width: 260px;
+    #limit_search_to_folder {
```
Comment (Brankub) [inline djangocms_admin_style/sass/partials/_changelist.scss]: do you have only IDs? Maybe you could use CSS classes?

File: djangocms_admin_style/sass/partials/_tables.scss [hunk 2]
```diff
@@ -19,8 +19,37 @@ table {
...
+        &.descending {
+            background: url('../img/sorting-icons.png') -5px -94px no-repeat;
+            &:hover {
+                background: url('../img/sorting-icons.png') -5px -115px no-repeat;
+            }
+        }
+        &.sortremove {
+            background: url('../img/sorting-icons.png') -4px -5px no-repeat;
+            &:hover {
+                background: url('../img/sorting-icons.png') -4px -26px no-repeat;
```
Comment (Brankub) [inline djangocms_admin_style/sass/partials/_tables.scss]: We use double quotes: https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/.csscomb.json#L9

File: djangocms_admin_style/sass/partials/_changelist.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/partials/_changelist.scss]: unfortunately not :( checked it already and cant change the template

**Comment by Brankub** [comment]
LGTM :+1:

**Pull request merged**